script.js

```diff
@@ -1,4 +1,401 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var numOfTimesUserWon = 0;
+var numOfTimesPcWon = 0;
+var numOfDraws = 0;
+var numOfGamesPlayed = 0;
+var pcWinningPercent = 0;
+var userWinningPercent = 0;
+var userName = "";
+var gameMode = "Waiting for UserName";
+var gameVersion = "Normal Version";
+var reverseCount = 0;
+var recentWinner = "";
+
+var main = function (userInput) {
+  var myOutputValue = "";
+
+  // If game mode is "Waiting for UserName", store input as user's name into a variable
+  if (gameMode == "Waiting for UserName") {
+    userName = userInput;
+    myOutputValue =
+      `Welcome ${userName}! 👋<br ><br >` +
+      `Select the version that you would like to play by submitting "Normal" or "Korean".`;
```

Comment:
If im not wrong, you used the + sign to force a linebreak right? I'm not sure if <br> works in template literals. You might need to use \n instead if it doesnt work.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings